### PR TITLE
Documentation and examples

### PR DESCRIPTION
**Pull request by VladDubrovskis**
/cc @adgad @andrewgeorgiou1981 

Updated docs to clarify how to initialise the library as well as a more detailed reference to possible slot / library configurations.

Have a look and let me know if anything is missing or something is unclear.

Also please verify the details about the responsive sizes and formats are correct.

### CODE REVIEW

**Comment by VladDubrovskis** [comment]
Also have a look at the links to other documents from README.md (the link to full slot reference wont work) as they point to master branch. Not sure if there is any smart way we can make full links per branch?

File: README.md [hunk 1]
```diff
@@ -28,7 +28,20 @@ Include o-ads in the build and and add the following markup to the page:
 On initialisation o-ads will request a 300x250 (see Formats below) advert targeted from Google's test network.
 
 ### Initialization
-The o-ads `init()` method takes a JSON object as it's only argument. The configuration object sets various properties on o-ads.
+
```
Comment (adgad) [inline README.md]: Is this supposed to be ##?

File: docs/SLOT_CONFIG_REFERENCE.md [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+<div data-o-ads-label="false"></div>
+<div data-o-ads-label="true"></div>
+<div data-o-ads-label="left"></div>
+<div data-o-ads-label="right"></div>
+```
+
+---------------------------------------
+
+### Out of page
+Define the parameter if the slot to be requested should be out of page one. Default is `false`.
```
Comment (adgad) [inline docs/SLOT_CONFIG_REFERENCE.md]: Maybe worth a line saying what an out of page slot is.

File: docs/SLOT_CONFIG_REFERENCE.md [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+* Leaderboard `728x90`
+* SuperLeaderboard `970x90` or `970x66`
+* HalfPage `300x600`
+* Billboard `970x250`
+* Portrait `300x1050`
+* AdhesionBanner `320x50`
+* MicroBar `88x31`
+* Button2 `120x60`
+
+***Note:*** Formats can only be configured via JSON configuration. Also once added, they can be used a number of slots alongside the default formats.
```
Comment (adgad) [inline docs/SLOT_CONFIG_REFERENCE.md]: This is only on the global json config right (not per slot...)?

**Pull request merged**